rules/integrations/aws/persistence_ec2_route_table_modified_or_deleted.toml

```diff
@@ -0,0 +1,143 @@
+[metadata]
+creation_date = "2021/06/05"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2025/01/27"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies AWS CloudTrail events where an EC2 route table or association has been modified or deleted. Route table or
+association modifications can be used by attackers to disrupt network traffic, reroute communications, or maintain
+persistence in a compromised environment. This is a [New
+Terms](https://www.elastic.co/guide/en/security/master/rules-ui-create.html#create-new-terms-rule) rule that detects the
+first instance of this behavior by the `aws.cloudtrail.user_identity.arn` field in the last 10 days.
+"""
+false_positives = [
+    """
+    Route Table could be modified or deleted by a system administrator. Verify whether the user identity, user agent,
+    and/or hostname should be making changes in your environment. Route Table being modified from unfamiliar users
+    should be investigated. If known behavior is causing false positives, it can be exempted from the rule. Also
+    automated processes that use Terraform may lead to false positives.
+    """,
+]
+from = "now-9m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS EC2 Route Table Modified or Deleted"
```

Comment:
Not a new rule, name and file name adjusted, investigation guide added and logic adjusted.